config/always-sed.m4

```diff
@@ -2,15 +2,15 @@ dnl #
 dnl # Set the flags used for sed in-place edits.
 dnl #
 AC_DEFUN([ZFS_AC_CONFIG_ALWAYS_SED], [
-	ac_inplace=""
-	AC_CACHE_CHECK([for sed --in-place], [ac_cv_path_SED],
-		[AC_PATH_PROGS_FEATURE_CHECK([SED], [sed],
-			[[tmpfile=$(mktemp)
-			  echo foo > $tmpfile
-			  $ac_path_SED --in-place 's#foo#bar#' $tmpfile \
-			  && ac_cv_path_SED=$ac_path_SED
-			  rm $tmpfile]],
-			[ac_inplace="-i ''"])])
-	AS_IF([test "x$ac_inplace" = "x"], [ac_inplace="--in-place"])
-	AC_SUBST([ac_inplace])
+	AC_REQUIRE([AC_PROG_SED])dnl
```

Comment:
The autoconf manual suggested adding dnl to avoid empty lines.

https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Prerequisite-Macros.html

> You are furthermore encouraged, although not required, to put all AC_REQUIRE calls at the beginning of a macro. You can use dnl to avoid the empty lines they leave.

Reply:
Although there are a ton of blank lines anyway in configure, so there's not much advantage here. I can remove it to make the source look better?